suse_openstack_cloud

```diff
@@ -395,14 +395,14 @@ fi
 ###############################################################
 section_header "RabbitMQ broker"
 
-if [ -e /usr/sbin/rabbitmqctl ]; then
-    rabbitmq_tmpdir=$(mktemp -d)
-    rabbitmq_report_log=$rabbitmq_tmpdir/rabbitmq_report.log
+rabbitmqctl=/usr/lib64/rabbitmq/bin/rabbitmqctl
```

Comment:
And why don't we simply run the rabbitmqctl from /usr/sbin? That would allow not having to switch directory and is more, hrm, intuitive :-)